schematic/manifest/generator.py

```diff
@@ -1635,6 +1635,7 @@ def create_single_manifest(
     def create_manifests(
         path_to_data_model: str,
         data_types: list,
+        data_model_labels: str = "class_label",
         access_token: Optional[str] = None,
```

Comment:
Can we change this to
`data_model_labels: Literal["class_label", "display_label"] = "class_label",`